pom.xml

```diff
@@ -36,7 +36,7 @@ A build status publisher that notifies channels on a IRC server
         <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
         <version.instant-messaging.plugin>1.47</version.instant-messaging.plugin>
         <java.level>8</java.level>
-        <jenkins.version>2.289.1</jenkins.version>
```

Comment:
@basil : Regarding non-LTS requirements: would this change work with 2.319.2 as the latest LTS today? or are there some required core changes starting from 2.320 that were not backported? (This might be behind stack traces I've seen, I modified the pom.xml to run on 2.319.2 for that test env and it compiled without issues at least...)

Reply:
> This might be behind stack traces I've seen, I modified the pom.xml to run on 2.319.2 for that test env and it compiled without issues at least...

In other words, you didn't test this PR, but rather your own ill-conceived alternative 🤣

Good thing I didn't waste any time debugging that stack trace, especially without the ability to see your code 😉

The Guava upgrade was not backported. It will be delivered in the March LTS.

Reply:
Apparently yes, discovered in the depths of hindsight. Using a N-1 core, what could possibly go wrong... So that should be it, right?